presubmit.py

```diff
@@ -210,6 +210,13 @@ def lint(paths: List[Path]) -> bool:
 def pytype(paths: List[Path]) -> bool:
     """Run pytype on |path| if it is a python file. Return False if it fails
     type checking."""
+    # Check that we are on Python3.7.
+    if not sys.version.startswith('3.7'):
+        logs.error(
+            'Python version is: "%s". You should be using 3.7.'
+            'Not running pytype.', sys.version)
+        return False
```

Comment:
Returning False means presubmit will always fail. Not sure if that's the correct behavior if it should return True.

Reply:
i thought your intention here was to skip pytype on anything > 3.7. if that is case, then should return true.

Reply:
That is my intention. Not sure why it should return True though? They are using the wrong version so I'm reporting it as a failure since I can't check instead of reporting at as a success when I can't check.

Reply:
Just wanted to make my reasoning more clear but if you think it should return True I can change it.

Reply:
i mean does it work if you just type "make presubmit", it does fail on pytype, but still continues the other checks. i remember you implemented that way to do all tests. then it is fine.

Reply:
Sorry, top level question: what do we want to use 3.7 or 3.8?

Should we set the version we want for our venv? E.g., if we want to use 3.7, we can update the Makefile to initialize the venv with:
```
${VENV_ACTIVATE}: requirements.txt
	python3.7 -m venv .venv
```
which should ensure that the presubmit runs with the right version.

Reply:
I think offline we (Abhishek and I) decided we didn't want to force a Python minor version on users.

Reply:
That's fine, but it'd still be useful to decide which one is our chosen/preferred/recommended version and indicate that, so people can know with which version they can expect to have most things working. E.g., what is more important: to be able to use 3.8 features or pytype. If we prefer say 3.7, in the Makefile we could eg try to initialize venv with python3.7 first, and if that fails use python3

Reply:
3.7 is the preferred one. I think having the makefile try python3.7 before python3 is a good idea.

Reply:
wanna add it in this pr?

Reply:
I'd rather land this and unbreak fuzzbench development on my machine and figure out how to improve this later.